### DISCUSSION TITLE
نمایش اعداد فارسی

### DISCUSSION THREAD

**Discussion by ghafouri-reza** [majidh1/JalaliDatePicker#47] [Question] [answered]
سلام
تشکر بابت کار بسیار عالیتون.
من در یک محیط وب از این datepicker استفاده کردم و اعداد رو به صورت انگلیسی نمایش میده .
ممکنه برای نمایش فارسی اعداد راهنمایی بفرمایید .

با سپاس

**Comment by majidh1** [comment, marked as answer]
سلام با تشکر از همکاری @HirbodBehnam عزیز این امکان اضافه شد میتونید از persianDigits در تنظیمات استفاده کنید
